docs/backend/content-types/index.md

```diff
@@ -12,15 +12,223 @@ myst:
+  />
+  <alias from="edit"
+         to="@@edit"
+  />
+  <alias from="sharing"
+         to="@@sharing"
+  />
+  <alias from="view"
+         to="(selected layout)"
+  />
+
+  <!-- Actions -->
+  <action action_id="view"
+          category="object"
+          condition_expr=""
+          icon_expr="string:toolbar-action/view"
+          title="View"
+          url_expr="string:${object_url}"
+          visible="True"
+          i18n:attributes="title"
+  >
+    <permission value="View" />
+  </action>
+  <action action_id="edit"
+          category="object"
+          condition_expr="not:object/@@plone_lock_info/is_locked_for_current_user|python:True"
+          icon_expr="string:toolbar-action/edit"
+          title="Edit"
+          url_expr="string:${object_url}/edit"
+          visible="True"
+          i18n:attributes="title"
+  >
+    <permission value="Modify portal content" />
+  </action>
+
+</object>
+```
+
+The `name` attribute on the root element in the XML must match the name in the filename and the name listed in `types.xml`.
+
+Set the `i18n:domain` to the i18n domain which includes translations for this content type. This is usually the same as the name of the Python package which contains the content type.
+
+The XML sets a number of FTI properties:
+
+`action` elements
+:   Defines additional {doc}`actions <portal-actions>` which are available for this content type.
+
+`add_permission`
+:   Id of the permission controlling whether the current user has permission to add this content type.
+
+`allow_discussion`
+:   Boolean.
+    Controls whether Plone's commenting system is enabled for this content type.
+
+`allowed_content_types`
+:   List of content types which can be added inside this one.
+    Only used if `filter_content_types` is True.
+
+`behaviors`
+:   List of {doc}`behaviors <behaviors>` enabled for this content type.
+
+`description`
+:   Short description displayed in the UI.
+
+`factory`
+:   Name of the factory adapter used to create new instances of the content type.
+    Usually the same as the content type name.
+
+`filter_content_types`
+:   Boolean.
+    Controls which content types can be added inside this one.
+    If `True`, allow only the types listed in `allowed_content_types`.
+    If `False`, allow any content type that the user has permission to add.
+
+`global_allow`
+:   Boolean.
+    Set to `True` to allow adding the content type anywhere in the site where the user has permission.
+    Set to `False` to only allow adding it inside other content types that include this one in `allowed_content_types`.
+
+`klass`
+:   Dotted path to the Python class for this content type.
+
+`model_file`
+:   Location of an XML file to load as the content type's schema.
+    This is an alternative to `schema` and `model_source`.
+
+`model_source`
+:   Inline XML schema for the content type.
+    This is an alternative to `schema` and `model_file`.
+
+`schema`
+:   Dotted path to the schema for this content type.
```

Comment:
Shall we mention, one of those must be set. Otherwise it fails silently. Event if only behaviors are used and no direct fields are needed.

Reply:
@jensens I updated it to clarify.